__tests__/app.test.js

```diff
@@ -13,38 +13,92 @@ beforeEach(() => {
+            "serves an array of all topics"
+          );
+          expect(response.body["GET /api/articles"].description).toEqual(
+            "serves an array of all articles"
+          );
+          expect(
+            response.body["GET /api/articles/:article_id"].description
+          ).toEqual("returns one article when given a valid id");
+        });
+    });
   });
-  test("200: all topics are returned", () => {
-    return request(app)
-      .get("/api/topics")
-      .expect(200)
-      .then((response) => {
-        const { topics } = response.body;
-        expect(topics).toHaveLength(3);
-        topics.forEach((topic) => {
-          expect(topic).toHaveProperty("slug", expect.any(String));
-          expect(topic).toHaveProperty("description", expect.any(String));
+
+  describe("GET getTopics", () => {
+    test("200: responds with a status 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+
+    test("200: all topics are returned", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const { topics } = response.body;
+          expect(topics).toHaveLength(3);
+          topics.forEach((topic) => {
+            expect(topic).toHaveProperty("slug", expect.any(String));
+            expect(topic).toHaveProperty("description", expect.any(String));
+          });
         });
-      });
+    });
   });
-  test("200: returns api documentaion", () => {
-    return request(app)
-      .get("/api")
-      .expect(200)
-      .then((response) => {
-        expect(response.body).toEqual(endpointDocumentation);
-        expect(response.body["GET /api"].description).toEqual(
-          "serves up a json representation of all the available endpoints of the api"
-        );
-        expect(response.body["GET /api/topics"].description).toEqual(
-          "serves an array of all topics"
-        );
-        expect(response.body["GET /api/articles"].description).toEqual(
-          "serves an array of all articles"
-        );
-      });
+
+  describe("GET getArticlesById", () => {
+    test("200: responds with the correct article object on the response body", () => {
+      return request(app)
+        .get("/api/articles/13")
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body;
+          expect(article).toEqual({
+            article_id: 13,
+            title: "Another article about Mitch",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "There will never be enough articles about Mitch!",
+            created_at: "2020-10-11T11:24:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+        });
+    });
+    test("200: for id: 13 responds with the correct article object on the response body", () => {
```

Comment:
This test feels unnecessary as you already tested the shape of the whole object, keys and values, in the previous test. If you wanted to keep this test, it would probably go first as it's more vague, checking data types, whereas the other test is interested in content